### PR TITLE
Adjustable V0 Bowden Strike Block

### PR DESCRIPTION
**Pull request by jgehrig**
This is a drop in replacement for the `Endstop_Strike_Block` used with Bowden extruders.

The endstop can be rotated to adjust the X gantry zero point. On my printer, the stock strike block was not large enough to trigger the homing switch before the toolhead crashed. This mod allows the users to more precisely set the zero point.

https://github.com/jgehrig/VoronUsers/tree/jg-endstop/printer_mods/jgehrig/v0_adj_bowden_endstop

### CODE REVIEW

**Comment by jgehrig** [comment]
The `printer_mods` `README.md` modifcation includes a line from #323.

That PR should bemerged first, so we don't get a merge conflict or bad link.

File: printer_mods/README.md [hunk 1]
```diff
@@ -137,7 +137,8 @@ like so:
 | | [4.3" Touchscreen Mount](./jeoje/4.3_Inch_Touchscreen_Mount) | Mounts a [Waveshare 4.3inch Capacitive Touch Display](https://www.waveshare.com/4.3inch-dsi-lcd.htm) to v2.4 and Switchwire Vorons |:x:|:x:|:heavy_check_mark:|:heavy_check_mark:|
 | | [Sturdy Handles](./jeoje/Sturdy_Handles) | Modified v2.2 handles with no panel rest and with 4 anchor points |:x: |:grey_question:|:heavy_check_mark:|:x:|
 | | [Molex MLX and Microfit 3.0 Bed Connector Mount](./jeoje/Molex_MLX_Microfit_Bed_Connector_Mount) | Solid mount your bed connectors |:x: |:x:|:heavy_check_mark:|:x:|
-| jgehrig | [V0 LCD Touchscreen Skirt](./jgehrig/v0_lcdskirt) | 3.2" LCD Touch Screen Skirt | :heavy_check_mark: | :x: | :x: | :x: |
+| jgehrig | [Adjustable Bowden Endstop](jgehrig/v0_adj_bowden_endstop) | Adjustable Bowden X-Axis endstop strike block | :heavy_check_mark: | :x: | :x: | :x: |
```
Comment (KiloQubit) [inline printer_mods/README.md:140]: Can you add an `:x:` for the Trident column on both lines?

File: printer_mods/jgehrig/v0_adj_bowden_endstop/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+1. Unscrew the stock endstop, being careful not to move the T-Nut.
+2. Loosely attach the modified endstop: `Adjustable_Bowden_Endstop`.
+3. Re-orient the toolhead limit switch so the lever is facing backwards. See picture.
+4. Spin the endstop until the limit switch triggers at the correct position.
+5. Firmly tighten the endstop screw.
+
+# CAD Design Files
+
+[Fusion 360 Web View](https://a360.co/3fExdho)
```
Comment (KiloQubit) [inline printer_mods/jgehrig/v0_adj_bowden_endstop/README.md:27]: this link goes to the view for the V0 display mount for me

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!  Please see fixes for README below.

**Comment by KiloQubit** [comment]
@jgehrig are you planning to update the PR to address the review?

**Comment by cbc02009** [comment]
Closing due to inactivity.

**Pull request closed without merging**